### PR TITLE
#35198: Log metrics for apps launched outside of registered command framework

### PR DESCRIPTION
**Pull request by josh-t**
This change logs the app version and one action metric for "export".

### CODE REVIEW

**Comment by manneohrstrom** [comment]
I think this is a preferred entry point: https://github.com/shotgunsoftware/tk-flame-review/blob/master/app.py#L101

File: app.py [hunk 1]
```diff
@@ -223,6 +223,17 @@ def populate_shotgun(self, session_id, info):
         
         # done!
         self._submission_done = True
+
+        # Log usage metrics
+        try:
+            self.log_metric("Version Create")
```
Comment (manneohrstrom) [inline app.py]: Not sure, but thinking maybe "Flame Sequence Export" is more illustrative? Or is that just repeating the app name?

**Comment by josh-t** [comment]
Cool. Will move the logging to the suggested entry point.

File: app.py (hunk 1 above)
Comment (josh-t) [inline app.py]: I think perhaps `Sequence Export`. That would log as `(tk-flame) tk-flame-review Sequence Export`. I'll update.

**Pull request closed without merging**